.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# 默认忽略的文件
```

Comment:
Please use English :)

Reply:
Sorry, I didn't notice that I open this project folder with IDEA mistakenly.